src/main/java/com/minecolonies/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -164,15 +173,12 @@ public void onWorldTick(TickEvent.WorldTickEvent event)
     {
         super.onWorldTick(event);
 
-        if (event.phase != TickEvent.Phase.END)
-        {
-            return;
-        }
+        if (event.phase != TickEvent.Phase.END) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)